README.md

```diff
@@ -103,15 +108,102 @@ Rlog is configured via the following environment variables:
                     is defined here AND a logfile is specified via
                     RLOG_LOG_FILE then the output is sent to both.
                     Default: Not set - meaning the output goes to stderr.
+* RLOG_CONF_FILE:   This is the only control variable, which is only allowed to
+                    appear as an environment variable. It cannot appear in a
+                    config file. If this variable is set then rlog looks for
+                    the config file at the specified location, which needs to
+                    be the absolute path of the file. If this variable is not
+                    defined, then rlog will look for the config file in
+                    "/etc/rlog/your-executable-name.conf".
+* RLOG_CONF_CHECK_INTERVAL: Number of seconds between checking whether the
+                    config file has changed. By default, this is set to 15
+                    seconds. This means that within 15 seconds a changed
+                    logging configuration will take effect. Note that this
+                    check is only performed when a log message is actually
+                    written. If the program does nothing or doesn't log
+                    messages, the config file won't be read. If there is no
+                    config file or it has been removed then the configuration
+                    from the environment variables is used. Set this value to
+                    0 order to switch off the regular log file checking. Be
+                    careful when setting it to 0 via the config file: rlog will
```

Comment:
Maybe this should be another setting that can only be set via environment variables. Would work well with the RLOG_CONF_FILE, which is the other. Basically, the conf-file related ones aren't allowed in the conf file.

That way, we can simply have a 'section' here, for settings that can be set in both env and file, and another section for settings that can only be set in env.